projects/components/src/action-menu/action-search.provider.spec.ts

```diff
@@ -0,0 +1,99 @@
+/*!
+ * Copyright 2020 VMware, Inc.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+import { MockTranslationService } from '@vcd/i18n';
+import { ActionItem } from '../common/interfaces/index';
+import { SpotlightSearchService } from '../spotlight-search/index';
+import { ActionSearchProvider } from './action-search.provider';
+
+interface HasActionSearchProvider {
+    actionSearchProvider: ActionSearchProvider<any, any>;
+}
+
+const MOCK_DATA: ActionItem<any, any>[] = [
```

Comment:
Better have this as a function so that it gets not modified accidentally (in the future when new test may be added)